integrations/snowplow/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Snowplow",
+  "version": "1.0.0",
+  "overview": "Forward Snowplow events as metrics to LaunchDarkly experiments.",
+  "description": "Use Snowplow with LaunchDarkly's Experimentation feature to forward the behavioral data you wish to use as metrics in your experiments.",
+  "author": "Snowplow",
+  "supportEmail": "support@launchdarkly.com",
+  "links": {
+    "site": "https://snowplow.io/",
+    "launchdarklyDocs": "https://docs.launchdarkly.com/integrations/snowplow",
+    "supportWebsite": "https://docs.snowplow.io/docs/destinations/forwarding-events/google-tag-manager-server-side/launchdarkly-tag-for-gtm-ss/",
+    "privacyPolicy": "https://launchdarkly.com/policies/privacy/"
+  },
+  "categories": ["data"],
+  "icons": {
+    "square": "assets/images/square.svg",
```

Comment:
@adatzer is there any way we can use just the design icon part of your logo for this SVG? When it loads up it renders the long very small. You can see the difference of your logo compared to Splunk's. 
![CleanShot 2023-06-27 at 08 42 49](https://github.com/launchdarkly/integration-framework/assets/10308834/32bed39c-07c8-47f3-a6ea-acd028725bb0)